src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -71,6 +71,31 @@
         }
     }
 
+    .bdl-ContentExplorerIncludeSubfolders {
+        display: inline-block;
+        width: 100%;
+        height: 8 * $bdl-grid-unit;
+        vertical-align: middle;
+
+        .bdl-ContentExplorerIncludeSubfolders-toggle {
+            display: block;
+            float: left;
```

Comment:
`float` is sort of an older solution. are we able to use flexbox with `justify-content: space-between;`?